src/main/java/io/github/apace100/origins/power/PowerType.java

```diff
@@ -35,6 +35,9 @@ public PowerType setHidden() {
         return this;
     }
 
+    public void onRemoved() {}
+    public void onLost() {}
```

Comment:
Why is this here? :D

Reply:
For some reason fabric was giving me an error telling me those default methods were required there and no other branch complained about it ever again.